USAA does not support hardware tokens or email

pconwell opened the pull request:
USAA only supports three 'levels' of security: normal username and password, username and password with an SMS OTP, or username with PIN+OTP generated by a symantec app.

pconwell committed:
USAA does not support hardware tokens or email

USAA only supports three 'levels' of security: normal username and password, username and password with an SMS OTP, or username with PIN+OTP generated by a symantec app.

Carlgo11 reviewed (changes requested):
Alright, thanks for the clarification @pconwell. 
Could you remove the lines with `No` as the value? Values are false by default and so it's not necessary to specify (plus it also breaks the build).

pconwell committed:
Confirmed USAA allows email 2FA but not hardware

Users must both enable "CyberCode Text" AND separately enable "Recovery Logon Credentials" for email and/or sms for email and sms 2FA respectively. Hardware tokens are not and never have been supported.
